src/RegistrationController.js

```diff
@@ -61,115 +61,75 @@ function (
   return BaseLoginController.extend({
     className: 'registration',
     initialize: function() {
+      var clientId = this.options.settings.get('registration.clientId');
       var Schema = RegistrationSchema.extend({
-        url: 'api/v1/schema'
+        url: this.options.settings.get('baseUrl')+'/api/v1/registration/'+clientId+'/form'
```

Comment:
We should check if clientId is defined.

Reply:
We have a check in another place that returns false if not defined. I can do the same here.